docs/src/CommutativeAlgebra/groebner_bases.md

```diff
@@ -183,25 +187,21 @@ still sense to speak of fully reduced remainders. However, even if we start from
     the above notation and the division algorithms extend naturally to $K[x]^p$ and $K[x]_>^p$, respectively.
 	
 The OSCAR functions discussed below compute standard representations and polynomial weak standard representations, respectively.
+In the global case, they always return fully reduced remainders.
 
-```@julia
-reduce(I::IdealGens, J::IdealGens; ordering::MonomialOrdering = default_ordering(base_ring(J)))
-```
-
-```@julia
-reduce(f::T, F::Vector{T}; ordering::MonomialOrdering = default_ordering(parent(f))) where {T <: MPolyElem}
-```
-
-```@julia
-reduce(F::Vector{T}, G::Vector{T}; ordering::MonomialOrdering = default_ordering(parent(F[1]))) where {T <: MPolyElem}
+```@docs
+reduce(g::T, F::Vector{T}; 
+	ordering::MonomialOrdering = default_ordering(parent(f))) where {T <: MPolyElem}
```

Comment:
typo? Where does 'f' come from? Should this be F[1]?

Reply:
@ederc  Can you please replace f by F[1] both in groebner.jl and groebner_basis.md?